definitions.go

```diff
@@ -55,3 +58,24 @@ func (c *Client) ListVhostDefinitions(vhost string) (p *ExportedDefinitions, err
 
 	return p, nil
 }
+
+//
+// POST /api/definitions
+//
+
+// UploadDefinitions uploads a set of definitions and returns an error indicating if the operation was a failure
+func (c *Client) UploadDefinitions(p *ExportedDefinitions) error {
```

Comment:
This could follow e.g. `DeclareQueue` and `DeclareExchange` and return a pair of `(res *http.Response, err error)`

Reply:
sure thanks for the feedback, will make the changes